recipes/pdb-tools/meta.yaml

```diff
@@ -0,0 +1,139 @@
+{% set name = "pdb-tools" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pdb-tools-{{ version }}.tar.gz
+  sha256: b76c4cd6304a15e545eff2737a76b71db31b881573e5ba3a93dba9a71a79653b
+
+build:
+  entry_points:
+    - pdb_gap=pdbtools.pdb_gap:main
+    - pdb_chainbows=pdbtools.pdb_chainbows:main
+    - pdb_selmodel=pdbtools.pdb_selmodel:main
+    - pdb_sort=pdbtools.pdb_sort:main
+    - pdb_splitchain=pdbtools.pdb_splitchain:main
+    - pdb_uniqname=pdbtools.pdb_uniqname:main
+    - pdb_selelem=pdbtools.pdb_selelem:main
+    - pdb_chain=pdbtools.pdb_chain:main
+    - pdb_reatom=pdbtools.pdb_reatom:main
+    - pdb_segxchain=pdbtools.pdb_segxchain:main
+    - pdb_delhetatm=pdbtools.pdb_delhetatm:main
+    - pdb_b=pdbtools.pdb_b:main
+    - pdb_delchain=pdbtools.pdb_delchain:main
+    - pdb_selseg=pdbtools.pdb_selseg:main
+    - pdb_chainxseg=pdbtools.pdb_chainxseg:main
+    - pdb_shiftres=pdbtools.pdb_shiftres:main
+    - pdb_delinsertion=pdbtools.pdb_delinsertion:main
+    - pdb_keepcoord=pdbtools.pdb_keepcoord:main
+    - pdb_seg=pdbtools.pdb_seg:main
+    - pdb_mkensemble=pdbtools.pdb_mkensemble:main
+    - __init__=pdbtools.__init__:main
+    - pdb_fromcif=pdbtools.pdb_fromcif:main
+    - pdb_merge=pdbtools.pdb_merge:main
+    - pdb_delresname=pdbtools.pdb_delresname:main
+    - pdb_fetch=pdbtools.pdb_fetch:main
+    - pdb_fixinsert=pdbtools.pdb_fixinsert:main
+    - pdb_delres=pdbtools.pdb_delres:main
+    - pdb_tofasta=pdbtools.pdb_tofasta:main
+    - pdb_tidy=pdbtools.pdb_tidy:main
+    - pdb_intersect=pdbtools.pdb_intersect:main
+    - pdb_wc=pdbtools.pdb_wc:main
+    - pdb_selaltloc=pdbtools.pdb_selaltloc:main
+    - pdb_delelem=pdbtools.pdb_delelem:main
+    - pdb_rplresname=pdbtools.pdb_rplresname:main
+    - pdb_rplchain=pdbtools.pdb_rplchain:main
+    - pdb_tocif=pdbtools.pdb_tocif:main
+    - pdb_element=pdbtools.pdb_element:main
+    - pdb_selhetatm=pdbtools.pdb_selhetatm:main
+    - pdb_selresname=pdbtools.pdb_selresname:main
+    - pdb_selchain=pdbtools.pdb_selchain:main
+    - pdb_validate=pdbtools.pdb_validate:main
+    - pdb_selres=pdbtools.pdb_selres:main
+    - pdb_splitmodel=pdbtools.pdb_splitmodel:main
+    - pdb_selatom=pdbtools.pdb_selatom:main
+    - pdb_reres=pdbtools.pdb_reres:main
+    - pdb_splitseg=pdbtools.pdb_splitseg:main
+    - pdb_occ=pdbtools.pdb_occ:main
+    - pdb_head=pdbtools.pdb_head:main
+    - pdb_chkensemble=pdbtools.pdb_chkensemble:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - pdbtools
+  commands:
+    - pip check
+    - test -f $PREFIX/bin/pdb_gap  # [unix] 
```

Comment:
This is probably an overkill. Testing if an entry point was created is replicating a test that should be in conda-build.

If we cannot test a CLI b/c of the complexity of the call we can leave it out. I'd test 1-2, easy to implement, calls instead.